DiagManager/CustomDiagnostics/AlwaysOn Basic Info/AlwaysOnDiagScript.sql

```diff
@@ -8,32 +8,48 @@ PRINT '==========================='
 PRINT 'Database Mirroring Endpoint'
 PRINT '==========================='
 PRINT ''
+PRINT '--AG_hadr_endpoints_principals--'
```

Comment:
Can you add spaces between the dashes and the name - just for consistency with most identifiers. Trying to get them all to be using spaces:

```suggestion
PRINT '-- AG_hadr_endpoints_principals --'
```